test/tests/functional/pbs_reservations.py

```diff
@@ -1123,3 +1123,128 @@ def test_reservations_with_expired_subjobs(self):
+        verify that the the 2 ASAP reservations with same starting time
+        doesn't crashes PBS daemons
+        """
+        self.common_config()
+        attrs = {ATTR_J: '1-5',
+                 'Resource_List.select': '2:ncpus=1',
+                 'Resource_List.walltime': '15',
+                 'Resource_List.place': 'vscatter'}
+        j = Job(PBSROOT_USER)
+        j.set_sleep_time(15)
+        j.set_attributes(attrs)
+        jid = self.server.submit(j)
+        subjid = []
+        for i in range(1, 6):
+            subjid.append(j.create_subjob_id(jid, i))
+        self.server.expect(JOB, {'job_state=R': 4}, count=True,
+                           extend='t', id=jid)
+        self.server.expect(JOB, {'job_state=Q': 1}, count=True,
+                           extend='t', id=jid)
+        now = int(time.time())
+        rid1 = self.submit_asap_reservation(PBSROOT_USER, jid)
+        rid1_q = rid1.split('.')[0]
+        j1 = Job(PBSROOT_USER)
+        j1.set_sleep_time(15)
+        j1.set_attributes(attrs)
+        jid2 = self.server.submit(j1)
+        subjid2 = []
+        for i in range(1, 6):
+            subjid2.append(j.create_subjob_id(jid2, i))
+        rid2 = self.submit_asap_reservation(PBSROOT_USER, jid2)
+        rid2_q = rid2.split('.')[0]
+
+        a = {'reserve_state': (MATCH_RE, "RESV_RUNNING|5")}
+        self.server.expect(RESV, a, id=rid1, offset=15)
+        self.server.expect(RESV, a, id=rid2)
+        self.server.expect(JOB, {'job_state=X': 4}, count=True,
+                           extend='xt', id=jid)
+        self.server.expect(JOB, {'job_state': 'R'}, subjid[4])
+        self.server.expect(JOB, {'job_state': 'R'}, subjid2[0])
+        # Wait for reservation to delete from server
+        resv_queue = [rid1_q, rid2_q]
+        for queue in resv_queue:
+            msg = "Que;" + queue + ";deleted at request of pbs_server@"
+            self.server.log_match(msg, starttime=now, interval=10)
+        jids = [jid, jid2]
+        for job in jids:
+            self.server.expect(JOB, 'queue', op=UNSET, id=job)
+        exp_attrib = {'job_state': 'F', 'substate': '91'}
+        jobs = [subjid[1], subjid2[1]]
+        for jid in jobs:
+            self.server.expect(JOB, exp_attrib, id=jid, extend='x')
+        self.server.isUp()
+        self.mom.isUp()
+        self.scheduler.isUp()
+
+    def test_standing_resv_with_multivnode_job_array(self):
+        """
+        Test multivnode job array with standing reservation
+        """
+        self.common_config()
+        if 'PBS_TZID' in self.conf:
+            tzone = self.conf['PBS_TZID']
+        elif 'PBS_TZID' in os.environ:
+            tzone = os.environ['PBS_TZID']
+        else:
+            self.logger.info('Missing timezone, using America/Los_Angeles')
+            tzone = 'America/Los_Angeles'
+
+        start = int(time.time()) + 10
+        end = int(time.time()) + 61
+        a = {'Resource_List.select': '2:ncpus=2',
```

Comment:
Since there are 2 vnodes with 4 cpus each, it may happen that both the chunks of this reservation get satisfied on the same vnode, in that case, it wouldn't be multi-vnode as you wanted it to be. Use place=vscatter